modules/mkdru_ding/js/mkdru_ding.theme.js

```diff
@@ -0,0 +1,256 @@
+      }
+      if (mkdru.settings.specific_subject_field) {
+        specific_subject_field = mkdru.settings.specific_subject_field + '=';
+      }
+    }
+
+    if (!link) {
+      link = choose_url(hit.location[0]);
+    }
+    var html = "";
+
+    html += '<li class="search-result" id="rec_' + hit.recid + '" ><div class="result-inner-wrapper">' +
+      '<h3 class="title">';
+    if (link) {
+      html += '<a href="' + link + '" target="_blank" >';
+    }
+    html += hit["md-title"];
+    if (link) {
+      html += '</a>';
+    }
+    if (hit.location[0]['md-medium']) {
+      html += " (" + hit.location[0]['md-medium'] + ")";
+    }
+    html += '</h3>';
+
+    // Add target to each result item.
+    var target = '';
+    if (typeof (hit.location[0]['@name']) !== 'undefined') {
+      target =  hit.location[0]['@name'];
+    }
+
+    html += '<div class="mkdru-target">' + target + '</div>';
+    html += '<div class="search-snippet-info">' +
+      '<p class="search-snippet"></p>' +
+      '<div class="ting-object clearfix">' +
+      '<div class="ting-overview clearfix">' +
+      '<div class="left-column left">' +
+      '<div class="picture">';
+
+    if (hit['md-thumburl']) {
+      html += '<img src="' + hit['md-thumburl'][0] + '" alt="' + hit['md-title'] + '" />';
+    }
+
+    html += '</div>' +
+      '</div>' +
+      '<div class="right-column left">';
+
+    if (hit["md-author"]) {
+      // expand on ; and reprint in the same form
+      var authors = hit["md-author"][0].split(';');
+      html += '<div class="creator"><span class="byline">' + Drupal.t('By') + ' </span>';
+      for (var i = 0; i < authors.length - 1; i++) {
+        html += '<a class="author" href="' + basePath + 'search/meta/' + specific_author_field + Drupal.theme.mkdruSafeTrim(authors[i]) + '">' + authors[i] + '</a> ;';
+      }
+      html += '<a class="author" href="' + basePath + 'search/meta/' + specific_author_field + Drupal.theme.mkdruSafeTrim(authors[authors.length - 1]) + '">' + authors[authors.length - 1] + '</a>';
+      if (hit['md-date']) {
+        html += '<span class="date"> (' + hit['md-date'] + ')</span>';
+      }
+      html += '</div><p></p>';
+    }
+    var dhit = hit.location[0];
+    if (dhit["md-journal-subpart"]) {
+      html += '<div class="mkdru-result-journal-subpart">' + dhit["md-journal-subpart"];
+      html += '</div><p/>';
+    } else if (hit["md-journal-title"]) {
+      html += '<div class="mkdru-result-journal">' + hit["md-journal-title"];
+      html += '</div><p/>';
+    }
+    if (dhit["md-subject"] && dhit["md-subject"].length > 0) {
+      var search_query;
+      html += '<div class="mkdru-result-subject"><p>';
+      for (var i = 0; i < dhit["md-subject"].length - 1; i++) {
+        search_query = 'su=&quot;' + specific_subject_field + dhit["md-subject"][i] + '&quot;';
+        html += '<a href="' + basePath + 'search/meta/' + search_query + '">' + dhit["md-subject"][i] + '</a> ';
```

Comment:
href needs Drupal.settings.basePath + Drupal.settings.pathPrefix to work consistently